docs/index.html

```diff
@@ -193,6 +193,19 @@ <h3>Diego Barrios</h3>
                 <a href="https://github.com/diegobahumada" target="_blank"><i class="fab fa-github"></i></a>
             </div>
             </div>        
+        <div class="card">
+            <h3>Cristopher Reyes</h3>
+            <p>
+                Estudiante de ing. ejecucion informatica de <em>segundo año</em> sin experiencia laboral.
+            </p>
+            <p>
+                Interesado en <em>Ciberseguridad, servidores, redes, linux, algoritmos, entre otros</em>
+            </p>
+            <div class="links">
+                <a href="www.linkedin.com/in/cristopher-reyes-villagra-0202382a0" target="_blank"><i class="fab fa-linkedin"></i></a>
+                <a https://github.com/crisp101" target="_blank"><i class="fab fa-github"></i></a>
```

Comment:
```suggestion
            <p>
                Estudiante de ing. ejecución informática de segundo año sin experiencia laboral.
            </p>
            <p>
                Interesado en Ciberseguridad, servidores, redes, linux, algoritmos, entre otros
            </p>
            <div class="links">
                <a href="www.linkedin.com/in/cristopher-reyes-villagra-0202382a0" target="_blank"><i class="fab fa-linkedin"></i></a>
                <a https://github.com/crisp101" target="_blank"><i class="fab fa-github"></i></a>
```

¡Muchas gracias por participar!, actualmente estamos utilizando el tag em solo para [Extranjerismos](https://www.rae.es/duda-linguistica/como-se-escriben-los-extranjerismos-en-un-texto-en-espanol) por lo tanto creo que estos cambios dejarían todo más bonito.

Como un pequeño nit (nitpick, o un arregla si quieres) podrías dejar el mensaje del commit haciendo referencia a ti, por ejemplo, `Presentación crisp101`